test/txRelay.js

```diff
@@ -67,7 +67,7 @@ async function signPayload(signingAddr, sendingAddr, txRelay, destinationAddress
    nonce = await txRelay.getNonce.call(signingAddr)
    //Tight packing, as Solidity sha3 does
    hashInput = txRelay.address + pad(nonce.toString('16')).slice(2)
-               + destinationAddress.slice(2) + data.slice(2)
+               + destinationAddress.slice(2) + data.slice(2) + pad('0').slice(2)
```

Comment:
Can we make this part of the input to the signPayload function, rather than hardcoding?